circle.yml

```diff
@@ -7,6 +7,8 @@ general:
 machine:
   node:
     version: 4.5.0
+  environment:
+    CXX: g++-4.9
```

Comment:
Oh. this is a new dep for the ajv update ? or unrelated ?

Reply:
nope, this one is for compiling with -std=c++11 :)

Reply:
yes but I meant: what is requiring that in Arsenal 6.4 ?

Reply:
a compilation issue with 'diskusage' :)

Reply:
ok, I guess the version was bumped and we're not depending on a fixed one, leading to this being un-caught till now

Reply:
absolutly no idea, just fixing ^^